sdk/python/runtime/main.go

```diff
@@ -13,11 +13,15 @@ func New(
 ) *PythonSdk {
 	return &PythonSdk{
 		SDKSourceDir: sdkSourceDir,
+		RequiredPaths: []string{
+			"**/pyproject.toml",
+		},
```

Comment:
Could maybe use the opposite of this, i.e., exclude sets of files. Each language as their own common .gitignore globs.

Reply:
The behavior of `include`/`exclude` (as we inherit from buildkit) is actually that if *anything* is provided in `include`, that results in everything else being discluded. So when we specify `include` we're saying ***only*** include those files. Which actually is exactly what we want here; ignore everything unless told not to 🙂